### PR TITLE
Add sdBoundingBox, ELSmootherStepS

### PR DESCRIPTION
**Pull request by cutesthypnotist**

### CODE REVIEW

File: ELDistanceFunctions.cginc [hunk 1]
```diff
@@ -129,6 +129,16 @@ float fBox2Cheap(float2 pos, float2 box)
 	return max(d.x, d.y);
 }
 
+// Makes a wireframe box at p with scale b and e is how much to hollow out.
+float sdBoundingBox( float3 p, float3 b, float e )
```
Comment (hakanai) [inline ELDistanceFunctions.cginc:133]: Some code style cleanup wanted on this function.
Also missing the newline after the closing brace.

File: ELMathUtilities.cginc [hunk 2]
```diff
@@ -121,4 +121,10 @@ float3 ELMod(float3 dividend, float3 divisor)
     return dividend - divisor * floor(dividend / divisor);
 }
 
+// https://en.wikipedia.org/wiki/Smoothstep#Variations
+float ELSmootherStep(float edge0, float edge1, float x) {
```
Comment (hakanai) [inline ELMathUtilities.cginc]: Brace to next line

File: ELMathUtilities.cginc [hunk 3]
```diff
@@ -121,4 +121,10 @@ float3 ELMod(float3 dividend, float3 divisor)
     return dividend - divisor * floor(dividend / divisor);
 }
 
+// https://en.wikipedia.org/wiki/Smoothstep#Variations
+float ELSmootherStep(float edge0, float edge1, float x) {
+    x = clamp((x - edge0) / (edge1 - edge0), 0.0, 1.0);
+    return x * x * x * (x * (x * 6. - 15.) + 10.);
```
Comment (hakanai) [inline ELMathUtilities.cginc]: Floats to normalised form (6.0, 15.0, 10.0)

**Review by hakanai — CHANGES_REQUESTED** [review]

File: ELDistanceFunctions.cginc (hunk 1 above)
Comment (cutesthypnotist) [reply ELDistanceFunctions.cginc:133]: added newline

File: ELMathUtilities.cginc (hunk 2 above)
Comment (cutesthypnotist) [reply ELMathUtilities.cginc]: done

File: ELMathUtilities.cginc (hunk 3 above)
Comment (cutesthypnotist) [reply ELMathUtilities.cginc]: done

**Review by hakanai — APPROVED** [review]

**Pull request merged**